src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ingroovy/StepInGroovyScript.java

```diff
@@ -0,0 +1,22 @@
+package org.jenkinsci.plugins.workflow.cps.steps.ingroovy;
+
+import org.jenkinsci.plugins.workflow.cps.CpsScript;
+import org.jenkinsci.plugins.workflow.cps.persistence.PersistIn;
+
+import java.io.IOException;
+
+import static org.jenkinsci.plugins.workflow.cps.persistence.PersistenceContext.*;
+
+/**
+ * Base class that defines contract for steps written in Groovy.
```

Comment:
So if we wanted to add more metadata that steps-in-groovy would expose (required context, etc), we'd add more methods here?